src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2122 @@
+        private static event EventHandler<SpecificEngineResultEventArgs> _specificEngineResult;
+        private static event EventHandler<EventArgs> _speechDetected;
+        private static event EventHandler<DialogRequestedEventArgs> _dialogRequested;
+        private static event EventHandler<PrivateDataSetEventArgs> _privateDataSet;
+        private static event EventHandler<PrivateDataRequestedEventArgs> _privateDataRequested;
+        private static event EventHandler<FeedbackAudioFormatEventArgs> _feedbackAudioFormat;
+        private static event EventHandler<FeedbackStreamingEventArgs> _feedbackStreaming;
+        private static event EventHandler<VcTtsStreamingEventArgs> _vcTtsStreaming;
+        private static event EventHandler<RecognitionResult> _recognitionResult;
+        private static event EventHandler<CurrentLanguageChangedEventArgs> _currentLanguageChanged;
+        private static event EventHandler<StateChangedEventArgs> _stateChanged;
+        private static event EventHandler<ServiceStateChangedEventArgs> _serviceStateChanged;
+        private static event EventHandler<ErrorOccurredEventArgs> _errorOccurred;
+        private static VcMgrAllResultCallback _allResultDelegate;
+        private static VcMgrPreResultCallback _preResultDelegate;
+        private static VcMgrSpecificEngineResultCallback _specificEngineResultDelegate;
+        private static VcMgrBeginSpeechDetectedCallback _beginSpeechDetectedDelegate;
+        private static VcMgrDialogRequestCallback _dialogRequestDelegate;
+        private static VcMgrPrivateDataSetCallback _privateDataSetDelegate;
+        private static VcMgrPrivateDataRequestedCallback _privateDataRequestedDelegate;
+        private static VcMgrFeedbackAudioFormatCallback _feedbackAudioFormatDelegate;
+        private static VcMgrFeedbackStreamingCallback _feedbackStreamingDelegate;
+        private static VcMgrVcTtsStreamingCallback _vcTtsStreamingDelegate;
+        private static VcMgrResultCallback s_resultDelegate;
+        private static VcMgrCurrentLanguageChangedCallback _languageDelegate;
+        private static VcMgrSupportedLanguageCallback _supportedLanguagesDelegate;
+        private static List<string> _supportedLanguages;
+        private static VcMgrStateChangedCallback _stateDelegate;
+        private static VcMgrServiceStateChangedCallback _serviceStateDelegate;
+        private static VcMgrErrorCallback _errorDelegate;
+
+        /// <summary>
+        /// Initialize voice control manager.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// http://tizen.org/privilege/recorder
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
```

Comment:
Please use separate `<privilege>...</privilege>` and `<feature>...</feature>` for each privilege and feature. It should be `<privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
<privilege>http://tizen.org/privilege/recorder</privilege>` and 
`<feature>http://tizen.org/feature/speech.control</feature>
<feature>http://tizen.org/feature/microphone</feature>`
Please update it for every method in this class.

Reply:
Fixed.